HW0/P6/P6B.py

```diff
@@ -16,16 +16,22 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(x/10.0) for x in range(-60,1,1)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
-
+        parallelStart = time.time()
+        pool.map(burnTime(t), range(N))
```

Comment:
I think Raphael is correct here. I believe the map function takes each element from the second argument and feeds it into the function. So we want to feed in [t, t, t, t, .... ,t ] into burnTime so that the sleep lasts t seconds. In this case, rather than feeding in [t, t, t .... t] you'd be feeding in [1, 2, 3.... N], which means that the parallel version is sleeping at varying times while the serial version is always sleeping for t second.